docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,99 @@
+"""
+Coriolis parameters over the UK
```

Comment:
Not a plural, just "parameter". However, "Coriolis frequency" might be more correct. Happy to have discussion on this.